apps/hash-graph/bench/Cargo.toml

```diff
@@ -13,6 +13,7 @@ autobenches = false
 [dev-dependencies]
 graph = { path = "../lib/graph" }
 graph-test-data = { path = "../tests/test_data" }
+temporal-versioning = { workspace = true }
```

Comment:
Of course, this is of personal taste, but I usually to `temporal-versioning.workspace = true` (seen in some other projects), to mimic `temporal-versioning = "0.1" or something similar.

Reply:
I don't really mind either way. Personally I think I like the `{}` notation more as it's more consistent with `path = `, but this could also be written as `.path`. Maybe we can change it globally at some point but I prefer not having it in a few places only.